functions/venue.js

```diff
@@ -835,7 +835,7 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
   const venueRef = venuesRef.docs && venuesRef.docs[0];
 
   if (venueRef) {
-    await venueRef.update(updated);
```

Comment:
This same syntax is used in a couple of places and it might be broken there too, can you please check?

Reply:
yep, updated it in the other 2 places b4cad48